scripts/install

```diff
@@ -139,22 +140,26 @@ fi
 # Create focus user
 yunohost user create $focus_user -f $focus_user -l $focus_user -m ${focus_user}@auth.$domain -p $focus_password -q 0
 
-# Configure metronome
-mv /etc/metronome/conf.d/$domain.cfg.lua /etc/metronome/conf.d/$domain.cfg.lua.back
-mv /etc/metronome/conf.d/auth.$domain.cfg.lua /etc/metronome/conf.d/auth.$domain.cfg.lua.back
-mv /etc/metronome/conf.d/conference.$domain.cfg.lua /etc/metronome/conf.d/conference.$domain.cfg.lua.back
-mv /etc/metronome/conf.d/jitsi-videobridge.$domain.cfg.lua /etc/metronome/conf.d/jitsi-videobridge.$domain.cfg.lua.back
-mv /etc/metronome/conf.d/focus.$domain.cfg.lua /etc/metronome/conf.d/focus.$domain.cfg.lua.back
-
-metronome="/etc/metronome/conf.d/$domain.cfg.lua"
-cp ../conf/metronome.cfg.lua "$metronome"
-ynh_replace_string --match_string="__DOMAIN__" --replace_string="$domain" --target_file="$metronome"
-ynh_replace_string --match_string="__APP__" --replace_string="$app" --target_file="$metronome"
-ynh_replace_string --match_string="__VIDEOBRIDGE_SECRET__" --replace_string="$videobridge_secret" --target_file="$metronome"
-ynh_replace_string --match_string="__FOCUS_SECRET__" --replace_string="$focus_secret" --target_file="$metronome"
-ynh_replace_string --match_string="__PORT_COMPONENT__" --replace_string="$port_component" --target_file="$metronome"
-chown -R metronome:metronome "/etc/metronome/conf.d/"
-
+# Add Metronome domain conf template
+metronome_conf="/usr/share/yunohost/templates/jitsi/$domain.cfg.lua"
+mkdir -p /usr/share/yunohost/templates/jitsi/
+cp ../conf/metronome.cfg.lua $metronome_conf
+ynh_replace_string --match_string="__DOMAIN__" --replace_string="$domain" --target_file="$metronome_conf"
+ynh_replace_string --match_string="__APP__" --replace_string="$app" --target_file="$metronome_conf"
+ynh_replace_string --match_string="__VIDEOBRIDGE_SECRET__" --replace_string="$videobridge_secret" --target_file="$metronome_conf"
+ynh_replace_string --match_string="__FOCUS_SECRET__" --replace_string="$focus_secret" --target_file="$metronome_conf"
+ynh_replace_string --match_string="__PORT_COMPONENT__" --replace_string="$port_component" --target_file="$metronome_conf"
+
+touch "/usr/share/yunohost/templates/jitsi/auth.$domain.cfg.lua"
+touch "/usr/share/yunohost/templates/jitsi/conference.$domain.cfg.lua"
+touch "/usr/share/yunohost/templates/jitsi/jitsi-videobridge.$domain.cfg.lua"
+touch "/usr/share/yunohost/templates/jitsi/focus.$domain.cfg.lua"
```

Comment:
Not sure why you touch these o.O ? Is this really what you mean to do ?

Reply:
the touch of the files is to create empty files that will replace the metronome config files created when adding the differents sub-domains